src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -74,9 +90,11 @@ private static void Main(string[] args)
         // Issues a GET request to the Bridge to determine whether it is alive.
         // A return of 'true' means the Bridge is healthy.  A return of 'false'
         // indicates the Bridge is not healthy, and 'errorMessage' describes the problem.
-        private static bool PingBridge(string host, int port, out string errorMessage)
+        // A healthy response will also populate properties with the current Bridge configuration.
+        private static bool PingBridge(string host, int port, out string errorMessage, out Dictionary<string, string> properties)
```

Comment:
I think I'd prefer to keep it.  It was an omission in the original design to have no way for a Bridge to obtain the BridgeConfiguration of a Bridge that was already running.  The content already exists in the response, so rather than throw it away, we at least capture it.   It was cheap to capture and will be useful in other scenarios.